velox/exec/WindowFunction.cpp

```diff
@@ -73,18 +73,25 @@ std::unique_ptr<WindowFunction> WindowFunction::create(
   VELOX_USER_FAIL("Window function not registered: {}", name);
 }
 
-void WindowFunction::setNullEmptyFramesResults(
+void WindowFunction::setEmptyFramesResult(
     const SelectivityVector& validRows,
     vector_size_t resultOffset,
-    const VectorPtr& result) {
+    const VectorPtr& result,
+    const VectorPtr defaultValue) {
```

Comment:
Nit : call this defaultResult.

Result should be the last parameter of this function since its for writes. All parameters for reading should be ahead of it.

Reply:
Reverted, this was kept as the last parameter since a default value was provided.